### ISSUE TITLE
Build with CMake < 3.24 and openMP fails

### ISSUE THREAD

**Issue by MarDiehl** [DrTimothyAldenDavis/SuiteSparse#669]
**Describe the bug**
Building with CMake < 3.24 fails because
```
find_package ( OpenMP GLOBAL )
```
in `SuiteSparse_config/CMakeLists.txt` which has `cmake_minimum_required ( VERSION 3.22 )` but according to https://cmake.org/cmake/help/latest/command/find_package.html 3.24 is required.

3.24 is about 1.5 years old (https://www.kitware.com/cmake-3-24-0-is-available-for-download/)

**Screenshots**
```
CMake Error at SuiteSparse_config/CMakeLists.txt:73 (find_package):
  find_package called with invalid argument "GLOBAL"
```

**Desktop (please complete the following information):**
`linux-pkgs-dbg-ftn-interfaces` configuration on GitLab CI of PETSc (https://gitlab.com/petsc/petsc/-/jobs/5840470517) 

**Additional context**
https://gitlab.com/petsc/petsc/-/merge_requests

**Comment by mmuetzel** [comment]
Thanks for reporting.
#670 should work around that issue with older CMake versions.

**Comment by DrTimothyAldenDavis** [comment]
Would it be helpful to post a SuiteSparse 7.5.0.betaX release with this update?

**Comment by DrTimothyAldenDavis** [comment]
I've posted a 7.5.0.beta3 with this fix:  https://github.com/DrTimothyAldenDavis/SuiteSparse/releases/tag/v7.5.0.beta3

**Comment by DrTimothyAldenDavis** [comment]
Fixed in 7.5.0 (to be released today).
